README.md

```diff
@@ -65,6 +65,9 @@ The configuration options below are supported as command line arguments in the f
 $ node -r dotenv/config your_script.js dotenv_config_path=/custom/path/to/your/env/vars
 ```
 
+Note, some IDEs like Webstorm and IntellJ might have trouble properly passing `dotenv_config_path` to node. This does not impact command line use, just when configuring run options within the IDE. 
```

Comment:
Let’s scrap this IDE-specific stuff and instead expand the existing docs to explain how to use new env vars. Examples outlined [here](
https://github.com/motdotla/dotenv/issues/351#issuecomment-443790144) would be a good start

Reply:
I think it would be helpful for those with IDE problems to mention something like, "If your IDE is having trouble passing the CLI config arguments like `dotenv_config_path` you can use environment variables instead (linked to that section)". It will save someone time, I'm sure. We don't have to mention any specific IDE this way. What do you think?

Reply:
It would be better to take IDE-specific questions to StackOverflow or other support forums. Write a blog post about how you fixed your issue. Those help people the most.

Reply:
OK, I'll do that and make the readme more generic